sdk/communication/Azure.Communication.JobRouter/tests/RouterClients/ClassificationPolicyLiveTests.cs

```diff
@@ -243,6 +245,33 @@ public async Task CreateWorkerRequirementsClassificationPolicyTest()
             Assert.AreEqual(1, getClassificationPolicyResponse.Value.WorkerSelectors.Count);
         }
 
+        [Test]
+        public async Task CreateClassificationPolicyAndRemoveAProperty()
+        {
+            RouterAdministrationClient routerClient = CreateRouterAdministrationClientWithConnectionString();
+
+            var classificationPolicyId = GenerateUniqueId($"{nameof(CreateClassificationPolicyAndRemoveAProperty)}-ClassicationPolicy_w_WSelector");
+            var classificationPolicyName = $"Priority-ClassificationPolicy";
+            var workerSelectors = new List<WorkerSelectorAttachment>();
+            var labelSelectorAttachment = new StaticWorkerSelectorAttachment(new WorkerSelector("department", LabelOperator.Equal, new LabelValue("sales")));
+            workerSelectors.Add(labelSelectorAttachment);
+            var createClassificationPolicyResponse = await routerClient.CreateClassificationPolicyAsync(
+                new CreateClassificationPolicyOptions(classificationPolicyId)
+                {
+                    Name = classificationPolicyName,
+                    WorkerSelectors = workerSelectors
+                });
+
+            AddForCleanup(new Task(async () => await routerClient.DeleteClassificationPolicyAsync(classificationPolicyId)));
+
+            Assert.False(string.IsNullOrWhiteSpace(createClassificationPolicyResponse.Value.Name));
+
+            var updatedPolicyResponse = await routerClient.UpdateClassificationPolicyAsync(classificationPolicyId,
+                RequestContent.Create(new { Name = (string?)null }));
```

Comment:
Curious: is there a reason you need to cast null to `string?`?

Reply:
Oh, [I see why](https://learn.microsoft.com/en-us/dotnet/csharp/misc/cs0828).

When you have your arch board review, it would be good to show them this in a sample, to help further shape intended usage.

Reply:
I have added a sample code snippet showcasing this